src/main/java/hudson/plugins/ccm/CcmPublisher.java

```diff
@@ -57,26 +59,13 @@
     /** Default PMD pattern. */
     private static final String DEFAULT_PATTERN = "**/*.cs";
     /** Ant file-set pattern of files to work with. */
-    private final String pattern;
-    
+    private String pattern;
+
     @DataBoundConstructor
-    public CcmPublisher(final String healthy, final String unHealthy, final String thresholdLimit,
-            final String defaultEncoding, final boolean useDeltaValues,
-            final String unstableTotalAll, final String unstableTotalHigh, final String unstableTotalNormal, final String unstableTotalLow,
-            final String unstableNewAll, final String unstableNewHigh, final String unstableNewNormal, final String unstableNewLow,
-            final String failedTotalAll, final String failedTotalHigh, final String failedTotalNormal, final String failedTotalLow,
-            final String failedNewAll, final String failedNewHigh, final String failedNewNormal, final String failedNewLow,
-            final boolean canRunOnFailed, final boolean shouldDetectModules, final boolean canComputeNew,
-            final String pattern) {
-        super(healthy, unHealthy, thresholdLimit, defaultEncoding, useDeltaValues,
-                unstableTotalAll, unstableTotalHigh, unstableTotalNormal, unstableTotalLow,
-                unstableNewAll, unstableNewHigh, unstableNewNormal, unstableNewLow,
-                failedTotalAll, failedTotalHigh, failedTotalNormal, failedTotalLow,
-                failedNewAll, failedNewHigh, failedNewNormal, failedNewLow,
-                canRunOnFailed, shouldDetectModules, canComputeNew, PLUGIN_NAME);
-        this.pattern = pattern;
+    public CcmPublisher() {
+        super(PLUGIN_NAME);
     }
-    
```

Comment:
Wut? haha, haven't looked at the analysis core, and to be honest the CCM migration to the analysis was a copy+paste from another plug-in.

Do you reckon it won't break existing jobs?